INSTALL.md

```diff
@@ -36,7 +36,7 @@ This is one of several ways to [install and configure golang](https://golang.org
 First, download the Golang archive to `/tmp/`, then extract the archive to `/usr/local`.
 
 ```
-$ export VERSION=1.12.6 OS=linux ARCH=amd64  # change this as you need
+$ export VERSION=1.13.0 OS=linux ARCH=amd64  # change this as you need
```

Comment:
This need to be `1.13` without the `0`, otherwise the download will fail.

```suggestion
$ export VERSION=1.13 OS=linux ARCH=amd64  # change this as you need
```